docs/lang/branch.md

```diff
@@ -1,79 +1,79 @@
-一个程序默认是按照代码的顺序执行下来的，有时我们需要选择性的执行某些语句，这时候就需要分支的功能来实现。选择合适的分支语句可以提高程序的效率。
+By default, a program is executed following the order of code. Sometimes when certain statements need to be selectively executed, we can use the the branch function to achieve such flow control. Choosing the appropriate branch statement can improve the efficiency of the program.
 
-## if 语句
+## if statement
 
-### 基本 if 语句
+### Basic if statement
 
-以下是基本 if 语句的结构。
+The following is the structure of a basic if statement.
 
 ```cpp
-if (条件) {
-  主体;
+if (condition) {
+  main body;
 }
 ```
 
-if 语句通过对条件进行求值，若结果为真（非 0），执行语句，否则不执行。
+The if statement calculates the condition to check: if the result is true (not 0), the statement is executed, otherwise it is not executed.
 
-如果主体中只有单个语句的话，花括号可以省略。
+If there is only a single statement in the body, the curly braces can be omitted.
 
-### if...else 语句
+### if...else statement
 
 ```cpp
-if (条件) {
-  主体1;
+if (condition) {
+  statement1;
 } else {
-  主体2;
+  statement2;
 }
 ```
 
-if...else 语句和 if 语句类似，else 不需要再写条件。当 if 语句的条件满足时会执行 if 里的语句，if 语句的条件不满足时会执行 else 里的语句。同样，当主体只有一条语句时，可以省略花括号。
+The if...else statement is similar to the if statement, and the else does not need to write more conditions. When the condition of the if statement is satisfied, the statement in the if will be executed. And when the condition of the if statement is not satisfied, the statement in the else will be executed. Similarly, when the main part has only one statement, the curly braces can be omitted.
```

Comment:
建议代码里的这些 if, else, break, switch 啥的用 mono （``） 包起来

Reply:
```suggestion
The if...else statement is similar to the if statement, and the else does not need to write more conditions. When the condition of the if statement is satisfied, the statement in the if will be executed. Otherwise, the statement in the else will be executed. Similarly, the curly braces can be omitted when the main part has only one statement.
```